src/ports/postgres/modules/kmeans/kmeans.py_in

```diff
@@ -34,6 +37,25 @@ def kmeans_validate_src(schema_madlib, rel_source, **kwargs):
 
 # ----------------------------------------------------------------------
 
+def kmeans_validate_expr(schema_madlib, rel_source, expr_point, **kwargs):
+    if not columns_exist_in_table(rel_source, [expr_point]):
```

Comment:
I think it'd be much clearer to make this if columns_exist... and return immediately if true.